src/casync-tool.c

```diff
@@ -71,6 +71,9 @@ static bool arg_mkdir = true;
 static CaDigestType arg_digest = CA_DIGEST_DEFAULT;
 static CaCompressionType arg_compression = CA_COMPRESSION_DEFAULT;
 
+static time_t extract_start_nsec;
+static time_t extract_end_nsec;
```

Comment:
wrong type... should be uint64_t

Reply:
indeed, fixed in the rework I just pushed.